lib/hawk/model/schema.rb

```diff
@@ -166,7 +166,7 @@ def to_s
         Caster.new(:float,    -> (value) { Float(value) }),
         Caster.new(:datetime, -> (value) { Time.parse(value) }),
         Caster.new(:date,     -> (value) { Date.parse(value) }),
-        Caster.new(:bignum,   -> (value) { BigDecimal.new(value) }),
+        Caster.new(:bignum,   -> (value) { BigDecimal(value) }),
```

Comment:
I've checked that changing from `BigDecimal.new()` to `BigDecimal(value)` also works on Ruby 1.9